File: tools/code_gen/go/output/TLog.go
func TLogGetLogLevel() TTkLogLevel {
  return TTkLogLevel(C.log_get_log_level());
}

func TLogSetLogLevel(log_level TTkLogLevel) TRet {
  return TRet(C.log_set_log_level((C.tk_log_level_t)(log_level)));
}

File: tools/code_gen/java/output/TLog.java
package awtk;


/**
 * log。
 *
 */
public class TLog { 

  /**
   * 获取log的级别。
   * 
   *
   * @return 返回log的级别。
   */
 public  static TTkLogLevel getLogLevel()  {
   return TTkLogLevel.from(log_get_log_level());
 }


  /**
   * 设置log的级别。
   * 
   * @param log_level log的级别。
   *
   * @return 返回RET_OK表示成功，否则表示失败。
   */
 public  static TRet setLogLevel(TTkLogLevel log_level)  {
   return TRet.from(log_set_log_level(log_level.value()));
 }

static private native int log_get_log_level();
static private native int log_set_log_level(int log_level);
};